api/src/main/java/org/pmiops/workbench/db/model/User.java

```diff
@@ -170,4 +173,31 @@ public Boolean getBlockscoreVerificationIsValid() {
   public void setBlockscoreVerificationIsValid(Boolean value) {
     blockscoreVerificationIsValid = value;
   }
+
+  @Column(name = "terms_of_service_completion_time")
+  public Timestamp getTermsOfServiceCompletionTime() {
```

Comment:
Should these things be populated on the `Profile` object in the swagger API definition? Is there some other way for the UI to infer which of these things have been completed on the client?

Reply:
Yeah, good point! Done.